middle-pgsql.cpp

```diff
@@ -1235,37 +1235,16 @@ void *middle_pgsql_t::pgsql_stop_one(void *arg)
     fprintf(stderr, "Stopping table: %s\n", table->name);
     pgsql_endCopy(table);
     time(&start);
-    if (!out_options->droptemp)
+    if (out_options->droptemp)
```

Comment:
I think the logic of `if --drop then drop table else if able to build an index then create index` is clearer than what we had before, and avoids a level of nested ifs.